pkg/kubectl/cmd/resource/get.go

```diff
@@ -341,7 +376,7 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 				updatePrintOptionsForOpenAPI(f, mapping, printOpts)
 			}
 
-			printer, err = f.PrinterForMapping(printOpts)
+			printer, err = f.PrinterForOptions(printOpts)
```

Comment:
@deads2k fyi. Had to change this as `NewVersionedPrinter` inside of PrinterForMapping was returning an error `no kind is registered for the type v1beta1.Table`.